training/heterogeneous-clusters/pt.grpc.sagemaker/code/requirements.txt

```diff
@@ -0,0 +1,2 @@
+torchvision
```

Comment:
Would like all prerequisites mentioned in main docs in the root of this project.